templates/email.hbs

```diff
@@ -1,5 +1,36 @@
 <html lang="nl-BE">
-	<body>
-		<h1>{{title}}</h1>
-	</body>
+<head>
+	<style>
+
+	</style>
+</head>
+<body>
+<header>
+	<h3>Persbericht {{creatorName}}</h3>
+	<hr>
+</header>
+
+<main>
+	<h1 class="press-release__title">{{title}}</h1>
+	<div class="press-release__content">{{htmlContent}}</div>
+</main>
+
+<footer>
+	<p>Wilt u geen e-mails meer van ons ontvangen? Jammer! <a href="#">Klik dan hier om u af te melden</a></p>
```

Comment:
In overleg met Alvin is beslist dat we geen support gaan bieden voor het (automatisch) uitschrijven van e-mails uit verzendlijsten.